homeassistant/components/sensibo/climate.py

```diff
@@ -368,3 +351,11 @@ async def async_assume_state(self, state) -> None:
             return
         await self._async_set_ac_state_property("on", state != HVAC_MODE_OFF, True)
         await self.coordinator.async_refresh()
+
+    async def async_write_result(
+        self, result: bool, attribute: str, state: Any
```

Comment:
Could you be more specific on the type hint here? Maybe like this:
```suggestion
        self, result: bool, attribute: str, state: str | int | bool
```